rockcraft/lifecycle.py

```diff
@@ -53,9 +53,9 @@ def run(command_name: str, parsed_args: "argparse.Namespace"):
 
     if not managed_mode and not destructive_mode:
         if command_name == "clean":
-            # TODO: support `rockcraft clean` for a environment
-            raise CraftError("`rockcraft clean` for an environment is not supported")
-        run_in_provider(project, command_name, parsed_args)
+            clean_provider(project_name=project.name, project_path=Path().absolute())
```

Comment:
so how does the code support cleaning [individual parts](https://github.com/canonical/rockcraft/pull/105/files#diff-3c4b0aa2b92800c658801083eb3665363665b72e13a25f1f16477389a06fe157R159)? Is that coming later?

Reply:
Coming later.  I added an [error and TODO](https://github.com/canonical/rockcraft/blob/bd878a14a58d466c0bb1512ae065aa2b89c3a9a9/rockcraft/lifecycle.py#L109) in the code and will track it internally with CRAFT-1395.